src/Propel/Generator/Platform/DefaultPlatform.php

```diff
@@ -1400,7 +1400,7 @@ public function normalizeTable(Table $table)
 
         foreach ($table->getColumns() as $column) {
             if ($column->getSize() && $defaultSize = $this->getDefaultTypeSize($column->getType())) {
-                if (intval($column->getSize()) === $defaultSize) {
+                if (null === $column->getScale() && intval($column->getSize()) === $defaultSize) {
```

Comment:
`getScale` can still return `null` while `getSize` returns the `defaultSize` and only then can the `size` be overriden to `null`